src/screens/kyc.screen.tsx

```
@@ -1138,6 +1139,30 @@ function Ident({ step, lang, onDone, onBack, onError }: EditProps): JSX.Element
     if (message.type === IframeMessageType) isStepDone(message as KycStepBase) ? onDone() : onBack();
   }
 
+  useEffect(() => {
+    if (step.type === (KycStepType as any).SUMSUB_VIDEO && step.session?.url) {
+      launchWebSdk(step.session.url);
+    }
+  }, [step]);
+
+  function launchWebSdk(accessToken: string) {
+    const snsWebSdkInstance = snsWebSdk
+      .init(accessToken, () => {
+        onError('Token expired');
+        return Promise.resolve('EXPIRED_ACCESS_TOKEN');
```

What is the returned string for? Maybe we should just throw an error instead of returning a random string? 😉

---

`onError` already induces some error handling on our end, so I'll simply return an empty string instead for now 👍